lib/ops/opsservice/configure.go

```diff
@@ -595,10 +596,13 @@ func (s *site) getPlanetMasterSecretsPackage(ctx *operationContext, p planetMast
 		return nil, trace.Wrap(err)
 	}
 
-	serviceSubnet, err := configure.ParseCIDR(p.serviceSubnetCIDR)
+	serviceSubnet, err := configure.ParseCIDR(p.serviceSubnet)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	// FIXME(dmitri): when letting kubernetes recreate the kubernetes service
```

Comment:
Why? Wasn't "kubernetes" service created by Kubernetes itself before too?

Reply:
Oh, I seem to have left this early comment which I should've removed. It is being created by kubernetes as before - will not change anything here.